### PR TITLE
Updated the resiliency module for Powerstore operator support in Installation Wizard UI

### PR DESCRIPTION
**Pull request by chaganti-rajitha**
# Description
This PR is to update and fix the defect raised for resiliency module in PowerStore driver operator support.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| (https://github.com/dell/csm/issues/885) |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [X] Have you tested whether the hyperlinks are working properly?
- [X] Did you add the examples wherever applicable?
- [X] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csminstallationwizard/src/index.html [hunk 1]
```diff
@@ -183,6 +189,18 @@
                           <label for="volumeless-pods" class="form-check-label ms-2 text-dark" data-bs-toggle="tooltip" data-bs-placement="right" title="Enable/Disable, if set to true will enable CSM for Resiliency to ignore pods without persistent volume attached to the pod.">Ignore Volumeless Pods</label>
                         </div>
                       </div>
+                      <div class="was-validated row mt-3 child-level-one" id="array-connectionloss-threshold">
+                        <label for="array-threshold" class="col-sm-3 col-form-label" data-bs-toggle="tooltip" data-bs-placement="right" title="Gives the number of failed connection polls that will be deemed to indicate array connectivity loss. Should not be set to less than 3.">Array Connectivity ConnectionLoss Threshold</label>
+                        <div class="col-sm-9"></div>
+                        <div class="col-sm-2">
+                          <input class="form-control input-lg" type="number" id="array-threshold" placeholder="3" oninput="validateInput(validateForm, CONSTANTS)" required>
```
Comment (niranjan-n1) [inline content/docs/deployment/csminstallationwizard/src/index.html:196]: Set the min value as 3 if it should not be reduced below 3

**Review by niranjan-n1 — CHANGES_REQUESTED** [review]
Minor change requested

**Review by samihan-dell — APPROVED** [review]

File: content/docs/deployment/csminstallationwizard/src/index.html [hunk 2]
```diff
@@ -183,6 +189,18 @@
                           <label for="volumeless-pods" class="form-check-label ms-2 text-dark" data-bs-toggle="tooltip" data-bs-placement="right" title="Enable/Disable, if set to true will enable CSM for Resiliency to ignore pods without persistent volume attached to the pod.">Ignore Volumeless Pods</label>
                         </div>
                       </div>
+                      <div class="was-validated row mt-3 child-level-one" id="array-connectionloss-threshold">
+                        <label for="array-threshold" class="col-sm-3 col-form-label" data-bs-toggle="tooltip" data-bs-placement="right" title="Gives the number of failed connection polls that will be deemed to indicate array connectivity loss. Should not be set to less than 3.">Array Connectivity ConnectionLoss Threshold</label>
```
Comment (sharont58) [inline content/docs/deployment/csminstallationwizard/src/index.html:193]: Gives the number of failed connection polls that will indicate array connectivity loss.

(remove "deemed to")

**Review by sharont58 — CHANGES_REQUESTED** [review]
One comment

**Review by sharont58 — DISMISSED** [review]

**Review by samihan-dell — APPROVED** [review]

**Review by niranjan-n1 — APPROVED** [review]

**Review by suryagupta4 — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**